src/sidebar/components/test/SelectionTabs-test.js

```diff
@@ -92,11 +92,16 @@ describe('SelectionTabs', () => {
       fakeStore.selectedTab.returns('note');
       const wrapper = createComponent();
 
-      const tabs = wrapper.find('button');
+      const annotationTabButton = wrapper
+        .find('Tab[label="Annotations"]')
+        .find('LabeledButton');
+      const noteTabButton = wrapper
+        .find('Tab[label="Page notes"]')
+        .find('LabeledButton');
 
-      assert.isTrue(tabs.at(1).hasClass('is-selected'));
-      assert.equal(tabs.at(1).prop('aria-selected'), 'true');
-      assert.equal(tabs.at(0).prop('aria-selected'), 'false');
+      assert.isTrue(noteTabButton.find('button').hasClass('is-selected'));
+      assert.equal(noteTabButton.prop('pressed'), true);
```

Comment:
Perhaps change to `assert.isTrue` and `assert.isFalse` for consistency across these three lines.

Reply:
Yes, that's nicer. Done.